src/ServiceControl.Audit/Infrastructure/Bootstrapper.cs

```diff
@@ -257,6 +266,8 @@ void RecordStartup(LoggingSettings loggingSettings, EndpointConfiguration endpoi
         TransportSettings transportSettings;
         TransportCustomization transportCustomization;
         static HttpClient httpClient;
+        static ILog metricsLog = LogManager.GetLogger("Metrics");
```

Comment:
Could we use some non-default logger or add application settings to make sure that the clients don't get performance metrics printed to the logs by default?

Reply:
Would that be covered by having a configuration value to turn metrics on (defaults to off)? Given that we have fallback systems to get config values from environment variables, it would allow us to turn it on by default in our development environments.

Reply:
I added a config setting `PrintMetrics` for this